### PR TITLE
PV2-3250 - Added AgeAtStartOfLearning to EarningEvents profiles

### PR DESCRIPTION
**Pull request by LiamWright**
https://skillsfundingagency.atlassian.net/browse/PV2-3250

### CODE REVIEW

File: src/SFA.DAS.Payments.Audit.Database/Payments2/Tables/Payments2.EarningEvent.sql [hunk 1]
```diff
@@ -24,6 +24,7 @@
 	SfaContributionPercentage DECIMAL(15, 5) NULL,
 	IlrFileName  NVARCHAR(400) NULL,
 	EventType NVARCHAR(4000) NULL,
+	AgeAtStartOfLearning TINYINT NOT NULL,
```
Comment (David-B-Read) [inline src/SFA.DAS.Payments.Audit.Database/Payments2/Tables/Payments2.EarningEvent.sql:27]: Think this should be nullable if there is no default value in the field

File: src/SFA.DAS.Payments.Database/Payments2/Tables/Payments2.EarningEvent.sql [hunk 2]
```diff
@@ -24,6 +24,7 @@
 	SfaContributionPercentage DECIMAL(15, 5) NULL,
 	IlrFileName  NVARCHAR(400) NULL,
 	EventType NVARCHAR(4000) NULL,
+	AgeAtStartOfLearning TINYINT NOT NULL,
```
Comment (David-B-Read) [inline src/SFA.DAS.Payments.Database/Payments2/Tables/Payments2.EarningEvent.sql:27]: Same comment as per the Audit database table

Comment (LiamWright) [reply src/SFA.DAS.Payments.Database/Payments2/Tables/Payments2.EarningEvent.sql]: Added

File: src/SFA.DAS.Payments.Audit.Database/Payments2/Tables/Payments2.EarningEvent.sql (hunk 1 above)
Comment (LiamWright) [reply src/SFA.DAS.Payments.Audit.Database/Payments2/Tables/Payments2.EarningEvent.sql]: Added

**Review by David-B-Read — DISMISSED** [review]

**Review by owencampbell — DISMISSED** [review]

File: src/SFA.DAS.Payments.EarningEvents.Application.UnitTests/Mapping/MappingTest.cs [hunk 3]
```diff
@@ -441,6 +441,42 @@ public void Maps_LearningAim()
             earningEvent.LearningAim.Reference.Should().Be("ZPROG001");
         }
 
+        [Test]
+        public void Maps_AgeAtStartOfLearning_From_Oldest_LearnStartDate()
+        {
+            var earningEvent = Mapper.Instance.Map<IntermediateLearningAim, ApprenticeshipContractType2EarningEvent>(learningAim);
```
Comment (abroodie) [inline src/SFA.DAS.Payments.EarningEvents.Application.UnitTests/Mapping/MappingTest.cs:447]: Not sure this line is needed

**Review by abroodie — CHANGES_REQUESTED** [review]
Just one minor comment.

File: src/SFA.DAS.Payments.EarningEvents.Application.UnitTests/Mapping/MappingTest.cs (hunk 3 above)
Comment (LiamWright) [reply src/SFA.DAS.Payments.EarningEvents.Application.UnitTests/Mapping/MappingTest.cs]: Have removed in the latest commit

**Review by abroodie — DISMISSED** [review]
looks ok to me

**Review by owencampbell — DISMISSED** [review]

**Pull request merged**